gradle/scala.gradle

```diff
@@ -49,12 +49,18 @@ test {
 sourceSets {
     main {
         scala {
-            srcDir "src/main/scala_$scalaBaseVersion"
+            srcDirs += ["src/main/scala_$scalaBaseVersion", "src/main/java"]
+        }
+        java {
+            srcDirs = []
         }
     }
```

Comment:
@mmalohlava well with this setting all works fine even when classes are in appropriate directories (java/scala) but if you think this config might cause trouble then I don't see any problems in keeping everything in scala folder.